docker-compose/monitor/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: build
+build: clean-jaeger
+	cd ../../ && \
+	make build-all-in-one && \
+	make docker-images-jaeger-backend && \
+	docker tag jaegertracing/all-in-one:latest jaegertracing/all-in-one:dev
+
+.PHONY: run
```

Comment:
Intentionally named this `run` instead of `run-connector` to imply it's the default make target to use to bring up the system locally.